examples/step-49/doc/results.dox

```diff
@@ -328,3 +328,27 @@ the nice detail of the boundary fitted mesh if we refine two more times:
 
 <img src="https://www.dealii.org/images/steps/developer/step-49.yuhan.11.png"
      alt="" width="400" height="355">
+
+
+<h3> Possible extensions </h3>
+
+<h4> Modify a mesh </h4>
+
+Similar to the operations in step-1, iterate over a mesh and change boundary
+indicators.
```

Comment:
That's only a half-complete sentence -- it lacks a subject :-)

Can you scaffold this some more? What's the purpose of this possible extension? Why does this make for a good exercise? What would one *learn* or *practice* by doing so?